gpt_index/langchain_helpers/chain_wrapper.py

```diff
@@ -152,3 +152,39 @@ def last_token_usage(self) -> int:
     def last_token_usage(self, value: int) -> None:
         """Set the last token usage."""
         self._last_token_usage = value
+
+    async def _apredict(self, prompt: Prompt, **prompt_args: Any) -> str:
+        """Inner predict function.
+
+        If retry_on_throttling is true, we will retry on rate limit errors.
+
+        """
+        llm_chain = LLMChain(prompt=prompt.get_langchain_prompt(), llm=self._llm)
+
+        # Note: we don't pass formatted_prompt to llm_chain.predict because
+        # langchain does the same formatting under the hood
+        full_prompt_args = prompt.get_full_format_args(prompt_args)
+        # TODO: support retry on throttling
+        llm_prediction = await llm_chain.apredict(**full_prompt_args)
+        return llm_prediction
+
+    async def apredict(self, prompt: Prompt, **prompt_args: Any) -> Tuple[str, str]:
+        """Predict the answer to a query.
```

Comment:
ooc does this also get autogenerated in the docs? if so should we explicitly mention that it's async?

Reply:
Doesn't async dec automatically get tagged as async in the sphinx? 

But yes, I should make docstring more explicit as well.